engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -797,7 +798,7 @@ private void execute() throws YarnException, IOException
       }
 
       numTotalContainers += containerRequests.size();
```

Comment:
Okay, I can use some input for this then and we can discuss it since this is legacy code and like I said the variable is not needed in any decision making and is only used in log messages.